app/views/shared/_feed.html.erb

```diff
@@ -1,22 +1,79 @@
-
-<% @posts.each do |post| %>
-  <div>
-    <div class="row">
-      <div class="col-md-3"><span><%= gravatar_for post.user %></span></div>
-      <div class="col-md-9">
-        <span><%= post.user.username %>:</span>
-        <span><%= post.content %></span>
-        <p><%= time_ago_in_words(post.created_at) %> ago</p>
+<section class="card mb-3">
+  <% @posts.each do |post| %>
+    <div class="p-3">
+      <div class="row">
+        <div class="col-md-3"><span><%= gravatar_for post.user %></span></div>
+        <div class="col-md-9">
+          <span><%= post.user.username %>:</span>
+          <span><%= post.content %></span>
+          <p><%= time_ago_in_words(post.created_at) %> ago</p>
+        </div>
+        <div class="pl-3">
+          <% if liked?(post) %>
+            <%= link_to 'Unlike', post_like_path(post.id, id: like_post(post).id), method: :delete %>
+          <% else %>
+            <%= link_to 'Like', post_likes_path(post), method: :post %>
```

Comment:
This is looking for

```
irb(main):003:0> app.post_likes_path(1)
=> "/posts/1/likes"
```

and returning:

`The action 'index' could not be found for LikesController`

When I hit like.

Please fix it before moving on.